docker/Dockerfile

```diff
@@ -1,15 +1,14 @@
 FROM jupyter/datascience-notebook:latest
-MAINTAINER Serenata de Amor "datasciencebr@gmail.com"
 
 USER root
 
-RUN apt-get update && apt-get install -y \
-  unzip
+RUN apt-get update && apt-get install -y unzip
 
 USER jovyan
 
 COPY requirements.txt ./
 COPY conda_requirements.txt ./
+COPY config.ini.example ./config.ini
```

Comment:
The thing is (I guess) that either user have to manually copy `config.ini.example` or we set the `Dockerfile` to do it — because sooner or later a Python script will look for this file.

What do you think about it? Is there a _best practice_ within Docker community related t environment variables?